### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by tazz4843**

### CODE REVIEW

**Review by Geolykt — APPROVED** [review]
Nothing wrong

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**What is the goal of the feature?**
+A clear and concise description of what you want to happen.
```
Comment (0ddlyoko) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I think it's still an XY Problem
A sentence that looks like "Describe the goal of the feature" or "Describe what the feature should do" is more appropriated

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative features you've considered.
```
Comment (0ddlyoko) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Same here (maybe "Describe alternatives of this feature" or something better) ?

**Review by 0ddlyoko — CHANGES_REQUESTED** [review]
Some sentences that, for me, could be better ;)

**Review by 0ddlyoko — APPROVED** [review]

**Pull request merged**